### PR TITLE
Fix typo super->sb in method toString() of RemoteStoreNodeAttribute

### PR DESCRIPTION
**Pull request by dk2k**
Removed unused StringBuilder

### CODE REVIEW

**Comment by dblock** [comment]
Same comment as https://github.com/opensearch-project/OpenSearch/pull/15360#issuecomment-2307561923 please.

File: server/src/main/java/org/opensearch/node/remotestore/RemoteStoreNodeAttribute.java [hunk 1]
```diff
@@ -276,8 +276,6 @@ public boolean equals(Object o) {
 
     @Override
     public String toString() {
-        StringBuilder sb = new StringBuilder();
-        sb.append('{').append(this.repositoriesMetadata).append('}');
```
Comment (sandeshkr419) [inline server/src/main/java/org/opensearch/node/remotestore/RemoteStoreNodeAttribute.java:280]: I see @psychbot authored this. 

@psychbot - Was this string builder to be used in this method and it was a miss to not use it?

Comment (dbwiddis) [reply server/src/main/java/org/opensearch/node/remotestore/RemoteStoreNodeAttribute.java:280]: Pretty sure it was a typo in the next line ( super -> sb )

Comment (dk2k) [reply server/src/main/java/org/opensearch/node/remotestore/RemoteStoreNodeAttribute.java:289]: @dbwiddis I fixed the typo as per your suggestion

**Review by dbwiddis — APPROVED** [review]
LGTM!

**Comment by dbwiddis** [comment]
Created the backport.  @dk2k please help us out here; while you are correctly signing your commits with DCO (thank you) your GitHub commits are coming from a "noreply" address, and when we backport to the 2.x branch the DCO is failing due to the "Author address" mismatch.  Please update your GitHub profile to use the same address you're signing your DCO commits with.

**Pull request merged**